app/adapters/adapter.rb

```diff
@@ -18,14 +18,12 @@ def map_simple(model_field, public_field)
   # Register a one-to-one mapping of a date field.
   # @param [Symbol] model_field
   # @param [Symbol] public_field
-  # @param [String] date_format The Date format to use.
-  def map_date(model_field, public_field, date_format)
-    raise ArgumentError, "date_format cannot be nil" if date_format.nil?
+  def map_date(model_field, public_field)
     map_from_public public_field do |value|
       {model_field => time_from_public(value)}
     end
     map_to_public model_field do |value|
-      {public_field => value.strftime(date_format)}
+      {public_field => value.iso8601}
```

Comment:
Although there were no problems in the specs, possibly because they are using the same method, this may not be correct.  It might need to be modified to first ensure it is a UTC time, e.g.
```ruby
[21] pry(BagAdapter)> Time.now.iso8601
=> "2016-10-31T13:46:26-07:00"
[22] pry(BagAdapter)> Time.now.utc.iso8601
=> "2016-10-31T20:46:34Z"
```